src/ontology/mondo-edit.obo

```diff
@@ -574661,6 +574661,16 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7379" xsd:anyURI
 
+[Term]
+id: MONDO:1040001
+name: vertebral artery dissection
+def: "Vertebral artery dissection (VAD) is a rare cause of stroke in the general population; however, represents one of the more common causes of stroke in patients younger than 45 years of age. Its signs and symptoms can be vague, and diagnosis can be elusive. Spontaneous dissections have been reported. However, incidental minor trauma often precipitates this potentially dangerous condition. Often some neck distortion such as chiropractic manipulation, bending of the neck, or blunt trauma causes the dissection. The dissection of the artery may ultimately lead to a stroke which often can be delayed for days following the acute dissection." [PMID:28722857]
+xref: PMID:19096659
+xref: PMID:33659122
```

Comment:
xref should refer to an ID in another terminology (e.g. OMIM:##, Orphanet:###). Not PMID. 
PMID can be used as an annotation to a definition or synonym